cpp/src/io/csv/csv_reader.cu

```diff
@@ -61,6 +61,7 @@
 
 constexpr int32_t HASH_SEED = 33;
 constexpr size_t max_chunk_bytes = 64*1024*1024; // 64MB
+constexpr size_t max_row_bytes = 1024; // 1KB
 
```

Comment:
Is 1KB enough ? Unless we have a similar 1KB limit somewhere else, it doesn't seem to be a very large number for a large file with 100+ columns, and the mapping is per-page anyways, so I would think that 4KB or even 16KB might be a safer limit than 1KB (and relatively negligible compared to a typical chunk size).

Reply:
@OlivierNV What do you think about increasing the size with the number of columns? I'm thinking something like 4KB (one page) + 64*column_num should cover it well. 
I don't want to use a huge size for any column count, because then trimExtraRows() would get really slow when column count is small.

Reply:
That sounds reasonable to me